src/mgmt/srv_internal.h

```diff
@@ -98,6 +98,14 @@ int ds_mgmt_destroy_pool(uuid_t pool_uuid, const char *group, uint32_t force);
 void ds_mgmt_hdlr_pool_create(crt_rpc_t *rpc_req);
 void ds_mgmt_hdlr_pool_destroy(crt_rpc_t *rpc_req);
 int ds_mgmt_pool_get_acl(uuid_t pool_uuid, struct daos_acl **acl);
+/* list pools:
+ * Note: ds_mgmt_list_pools poolsp arg is a CaRT structure type
+ */
+void
+ds_mgmt_free_pool_list(struct mgmt_list_pools_one **poolsp, uint64_t len);
```

Comment:
Would you mind putting "void" and "ds_mgmt_free_pool_list" on the same line, and remove the empty line on line 108 (the new line numbers), please? Also, the comment could be together with the function implementation.

Reply:
The comment means, in other words, "mgmt_list_pools_one is defined in rpc.h"?